manifests/init.pp

```diff
@@ -75,6 +79,33 @@
     path => $certs::iop_advisor_engine::client_ca_cert,
   }
 
+  podman::quadlet { $service_name:
```

Comment:
You need to adjust https://github.com/ehelms/puppet-iop-advisor-engine/blob/master/spec/setup_acceptance_node.pp as those now conflict :)

Reply:
Yea... I wasn't sure how to mesh those two. I was hoping I'd just get an actual image to reference first :)